core/templates/pages/exploration-player-page/learner-experience/end-chapter-check-mark.component.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright 2022 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Component for the end chapter celebration check mark component.
+ */
+
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'oppia-end-chapter-check-mark',
+  template: require('./end-chapter-check-mark.component.html'),
+})
+export class EndChapterCheckMarkComponent {
+  constructor() {}
+  circleIsShown: boolean = false;
+  innerTickIsShown: boolean = false;
+
+  animateCheckMark(): void {
+    setTimeout(() => {
+      this.circleIsShown = true;
+    }, 500);
+    setTimeout(() => {
+      this.innerTickIsShown = true;
+    }, 750);
+    setTimeout(() => {
+      this.circleIsShown = false;
+    }, 2300);
+    setTimeout(() => {
+      this.innerTickIsShown = false;
+    }, 2550);
```

Comment:
Is there a way to implement this in the CSS?

Reply:
As discussed offline, can we use keyframes or transitions to implement this animation in the CSS?

Reply:
Done!